website/content/api-docs/system/managed-keys.mdx

```diff
@@ -132,6 +132,44 @@ $ curl \
   a read-write session to the HSM. This is a boolean expressed as a string (e.g.
   `"true"`). This key is mainly to work around a limitation within AWS's CloudHSM v5
   pkcs11 implementation.
+  
+#### AWS KMS Parameters
```

Comment:
Would be good to also provide details on how to create the key in KMS if the user wishes to do so.  e.g. with terraform, key_type of ESCSA and curve of P256 is customer_master_key_spec = "ECC_NIST_P256".  Also the key usage must be SIGN_VERIFY, not ENCRYPT_DECRYPT like seal users might expect.